### ISSUE TITLE
twweet-cli command doesn't work anymore.

### ISSUE THREAD

**Issue by CruiseDevice** [CruiseDevice/twweet-cli#41]
![twweet-cli-command-error](https://user-images.githubusercontent.com/10258150/30025925-65edd096-9198-11e7-9a23-5cba201db9c0.png)
twweet-cli command which was meant to run the application directly from anywhere instead of doing python3 twweet-cli.py does not work anymore. I have added the screenshot below of the error it is showing after running the command for the reference. @nithinswarrier can you please check it and confirm.

**Comment by 0x006E** [comment]
Yeah I will. For the moment I am not home.

**Comment by 0x006E** [comment]
Its not the command or the setup.py. I think you are missing the module ConfigReader which makes a runtime error and exits the app

**Comment by CruiseDevice** [comment]
@nithinswarrier The module is present but i think it is not importing. I checked this problem on the internet and i found that it has to do something with `__init__.py `file. https://stackoverflow.com/questions/4142151/how-to-import-the-class-within-the-same-directory-or-sub-directory I tried creating an empty `__init__.py` file as per the instructions given in the above link, but then the new issue comes.
